dockerfile/avalanche/Dockerfile

```
@@ -150,13 +144,11 @@ RUN for b in \
   tar \
   tee \
   tr \
+  vi \
   watch \
   which \
   ; do ln sh $b; done
```

In conjunction with https://github.com/strangelove-ventures/heighliner/pull/321/files#r2006357529, this should be
```suggestion
  sh \
  ; do ln ln $b; done
```